receptive.swagger.json

```diff
@@ -349,6 +349,35 @@
         }
       }
     },
+    "/comments": {
+      "get": {
+        "tags": [
+          "comment"
+        ],
+        "summary": "fetch Comment records",
+        "description": "get a list of Comment records",
+        "parameters": [
```

Comment:
it is also possible to pass `order_by : 'created_at'` in query params to order by creation time

Reply:
`created_at` is the only permitted order so not sure if that needs including.

Reply:
yes, youre are right, its still passed from frontend to the controller. might as well remove thatl